module/VuFind/config/module.config.php

```diff
@@ -799,6 +805,7 @@
     'Cart/Processor', 'Cart/Save', 'Cart/SearchResultsBulk',
     'Channels/Home', 'Channels/Record', 'Channels/Search',
     'Checkouts/History', 'Checkouts/PurgeHistory',
+    'Comments/UserList', 'Comments/DeleteComments',
```

Comment:
I'd reverse these for the sake of alphabetical order:

```suggestion
    'Comments/DeleteComments', 'Comments/UserList',
```